ur_simulation_ignition/launch/ur_moveit.launch.py

```diff
@@ -0,0 +1,377 @@
+# Copyright (c) 2021 PickNik, Inc.
```

Comment:
We should use file from the "Driver" repository. It will get merged here: UniversalRobots/Universal_Robots_ROS2_Driver#278

It would be great if you could also test this repos with that PR and give us a feedback there if it worked for you.

Reply:
I've tested the update-simplify-moveit-file branch. In general it works, but I had to do some modifications, because it doesn't work with the scaled_joint_trajectory_controller. The joint_trajectory_controller has to be set as default in the [controllers.yaml](https://github.com/benediktkreis/Universal_Robots_ROS2_Ignition_Simulation/blob/sim_time/ur_simulation_ignition/config/controllers.yaml). I guess the scaled one is not supported by the [ign_ros2_control](https://github.com/ignitionrobotics/ign_ros2_control) plugin. That is why I've copied the [ur_moveit.launch.py](https://github.com/benediktkreis/Universal_Robots_ROS2_Ignition_Simulation/blob/sim_time/ur_simulation_ignition/launch/ur_moveit.launch.py) and the [controllers.yaml](https://github.com/benediktkreis/Universal_Robots_ROS2_Ignition_Simulation/blob/sim_time/ur_simulation_ignition/config/controllers.yaml) with the one from your branch and modified them accordingly. Furthermore, without the robot_description_planning in [ur_moveit.launch.py](https://github.com/benediktkreis/Universal_Robots_ROS2_Ignition_Simulation/blob/sim_time/ur_simulation_ignition/launch/ur_moveit.launch.py) it isn't possible to plan a path using the drag function ("virtual ball") in RVIZ, I could just plan using predefined poses like "home", "up" and "test_configuration". Also, I don't know why the kinematics.yaml in [ur_moveit.launch.py](https://github.com/benediktkreis/Universal_Robots_ROS2_Ignition_Simulation/blob/sim_time/ur_simulation_ignition/launch/ur_moveit.launch.py) is loaded from a package called moveit_config_package, because it doesn't exist in my setup. That is why I renamed it to use the moveit_config_package instead.